cliquet/tests/test_authorization.py

```diff
@@ -73,6 +73,24 @@ def test_attributes_are_none_with_non_resource_requests(self):
         self.assertIsNone(context.resource_name)
         self.assertIsNone(context.check_permission)
 
+    def test_service_collection_path_is_correct(self):
```

Comment:
"is correct" is not explicit (https://plus.google.com/+JonathanLange/posts/YA3ThKWhSAj)

you could say : on creation, permission is bound to create on collection id